modules/host/host_test.go

```diff
@@ -40,7 +40,7 @@ func CreateHostTester(t *testing.T) (ht *HostTester) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	w, err := wallet.New(ct.State, tp, "")
+	w, err := wallet.New(ct.State, tp, "../../host_test.wallet")
```

Comment:
`rm -rf *.wallet` doesn't seem to catch wallets in nested directories.

good point. Looks like it didn't cause a problem here because there's only one test.